src/grt/src/fastroute/include/FastRoute.h

```diff
@@ -328,6 +323,13 @@ class FastRouteCore
 
   // maze3D functions
   void mazeRouteMSMDOrder3D(int expand, int ripupTHlb, int ripupTHub);
+  void addNbrPoints(int netID,
```

Comment:
Avoid abbreviations in the function names. Change `addNbrPoints` to `addNeighborPoints`.